src/GeniusYield/TxBuilder/Clb.hs

```diff
@@ -31,6 +31,7 @@ import           Control.Monad.Except
 import           Control.Monad.Random
```

Comment:
More updates to this module will follow in a different PR. Said PR will be able to unify privnet and clb code